cmd/utils/flags.go

```diff
@@ -2273,8 +2287,18 @@ func MakeChain(ctx *cli.Context, stack *node.Node, readonly bool) (*core.BlockCh
 	}
 	vmcfg := vm.Config{EnablePreimageRecording: ctx.Bool(VMEnableDebugFlag.Name)}
 
+	var overrides core.ChainOverrides
+	if ctx.IsSet(MaxTxsFlag.Name) {
+		overrides.MaxTxs = new(int)
+		*overrides.MaxTxs = ctx.Int(MaxTxsFlag.GetValue())
+	}
+	if ctx.IsSet(MaxCalldataFlag.Name) {
+		overrides.MaxCalldata = new(int)
+		*overrides.MaxCalldata = ctx.Int(MaxCalldataFlag.GetValue())
+	}
+
```

Comment:
When following the codes, it seems you don't have to set the flags here. Let me test codes without this and share the result.

Reply:
When I tested, it isn't needed!